### PR TITLE
Fix examples/http.js

### PR DESCRIPTION
**Pull request by LuKks**

### CODE REVIEW

**Comment by LuKks** [comment]
@mafintosh error seems unrelated

**Comment by mafintosh** [comment]
ya fixed in master

File: examples/http.js [hunk 1]
```diff
@@ -28,14 +28,14 @@ class ByteStream extends streamx.Readable {
     }
 
     if (this.byteOffset > 0) {
-      const [block, byteOffset] = await core.seek(this.byteOffset)
+      const [block, byteOffset] = await this.core.seek(this.byteOffset)
       this.byteOffset = 0
       this.index = block + 1
       this._select(this.index)
-      data = (await core.get(block)).slice(byteOffset)
+      data = (await this.core.get(block)).slice(byteOffset)
```
Comment (mafintosh) [inline examples/http.js:35]: can you change this to subarray? better that way

File: examples/http.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ async function start () {
   // hack until we update the replicator
   core.ready = (cb) => cb(null)
 
-  replicator(core, {
+  new Replicator(core, { // eslint-disable-line no-new
```
Comment (LuKks) [inline examples/http.js:87]: Replicator needed the `new` otherwise I would have leave it as it was

**Pull request merged**